src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -229,6 +229,9 @@ def load_arguments(self, _):
         with self.argument_context(scope + ' config ssl import') as c:
             c.argument('key_vault', help='The name or resource ID of the Key Vault')
             c.argument('key_vault_certificate_name', help='The name of the certificate in Key Vault')
+        with self.argument_context(scope + ' config ssl sync') as c:
+            c.argument('certificate_name', help='The name of the certificate to be synced')
```

Comment:
this can be certificate name or certificate resource ID?

Reply:
Actually why do you need the certName? can't we get the latest version of the cert from the AKV & just use that?